pyvista/plotting/plotting.py

```diff
@@ -1387,6 +1387,16 @@ def add_mesh(self, mesh, color=None, style=None, scalars=None,
             If set to ``True``, then the number of unique values in the scalar
             array will be used as the ``n_colors`` argument.
 
+        silhouette : dict, bool, optional
```

Comment:
This is mostly bikeshedding but I'd add a sentence about what the whole concept is, something along the lines of "Specify whether to plot a silhouette highlight for the mesh" (just to show what I mean).

Reply:
I tried something :sweat_smile: